client/src/components/AddChild/addchild.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from "react";
+import DeleteBtn from "../DeleteBtn";
+import Jumbotron from "../Jumbotron";
+import API from "../../utils/API";
+import { Link } from "react-router-dom";
+import { Col, Row, Container } from "../Grid";
+import { List, ListItem } from "../List";
+import { Input, TextArea, FormBtn } from "../Form";
+
+class AddChild extends Component {
+  state = {
+    name: "",
+    budget: 0,
+  };
+
+  handleInputChange = event => {
+    const { name, value } = event.target;
+    this.setState({
+      [name]: value
+    });
+  };
+
+  handleFormSubmit = event => {
+    event.preventDefault();
+    if (this.state.name && this.state.budget) {
+      API.getAddChild({
+        name: this.state.name,
+        budget: this.state.budget
+      })
+        // .then(res => this.loadJobs())
+        .catch(err => console.log(err));
+    }
+  };
+
+  render() {
+    return (
+      <Container fluid>
+        <Row>
+          <Col size="md-6">
+            <Jumbotron>
+              <h1>Create Child</h1>
+            </Jumbotron>
+            <form>
```

Comment:
You can probably extract out this form component to its own file and then import it in here, Your return for your render function is getting long, but the downside is having to pass down the props even more, but y'know it shouldn't be that bad.